lanl/docker-compose/configs/krakend.json

```diff
@@ -6,6 +6,36 @@
   "cache_ttl": "300s",
   "output_encoding": "json",
   "endpoints": [
+    {
+      "endpoint": "/v1/ochami/login",
+      "method": "POST",
+      "output_encoding": "json",
+      "backend": [
+        {
+          "url_pattern": "/hsm/v2/memberships", // need to replace with hydra route
+          "encoding": "json",
+          "sd": "static",
+          "host": [
+            "http://localhost:3000"
+          ]
+        }
+      ]
+    },
+    {
+      "endpoint": "/v1/ochami/physical/datacenters",
+      "method": "GET",
+      "output_encoding": "json",
+      "backend": [
+        {
+          "url_pattern": "/hsm/v2/memberships",
+          "encoding": "json",
+          "sd": "static",
+          "host": [
+            "http://smd:27779"
+          ]
+        }
+      ]
+    },
```

Comment:
Are there supposed to be duplicate `/v1/ochami/physical/datacenters` endpoints?

Reply:
Nope. That should be `/v1/ochami/physical/datacenters/{id}`. Not sure how that got changed.